handbook/marketing/README.md

```diff
@@ -116,6 +118,21 @@ Any changes to the current running ads visible to a user, including designs, key
 2. Compare existing ads against the newly proposed ad within the corresponding ad platform. ([Google Ads](https://ads.google.com/home/), [LinkedIn Campaign Manager](https://www.linkedin.com/campaignmanager/), etc.)
 3. If your change is approved, Field Marketer makes changes and creates a calendar reminder to check performance two weeks from the date changes were made.
 
+### Book an event
+For an event to be considered, booked, and scheduled, we follow the event issue template.
+
+1. Create a New GitHub Issue from the [#g-marketing board](https://app.zenhub.com/workspaces/g-marketing-64e6c8e2d35c7f001a457b7f/board).
+2. Select the "Event-preparation" template.
+3. Drag the issue into the "🗓 Ideas for future events" column as you work through the template.
+4. Once approval has been received, move the event into the "🗓 Planned events" column.
+5. If there was an issue created under the "📨New requests" from a team member, move any relevant information from the old issue to the new one and close the old one.
+
+### Review ongoing events
+In order to be timely, preparation is ongoing for events within the "🗓 Planned events" column within the [#g-marketing board](https://app.zenhub.com/workspaces/g-marketing-64e6c8e2d35c7f001a457b7f/board).
+
+1. Check "🗓 Planned events" and continue to work through steps in each event's issue.
+2. Remember to communicate with event organizers to ask questions when necessary.
```

Comment:
I'd cut this second step.  It's not an action you can take, it's a reminder . So it's not a step.  Either drop it out of the list, or in this case leave it out altogether  (I'd probably just do that)